src/core/src/bound_evaluate.cpp

```diff
@@ -281,17 +293,14 @@ bool ov::interval_bound_evaluator(const Node* node,
     auto low_1 = ov::evaluate_lower_bound(node->get_input_source_output(1));
     auto up_0 = ov::evaluate_upper_bound(node->get_input_source_output(0));
     auto up_1 = ov::evaluate_upper_bound(node->get_input_source_output(1));
+    if (!low_0 || !low_1 || !up_0 || !up_1)
+        return false;
```

Comment:
I still don't understand why can't we get the lowest value or greatest value if `low` or `up` is not specified.

Reply:
This fix does not correspond to evaluation calculations itself but to its comparator "misbehavior". Bounds evaluation algorithm is to be refactor in a separate story.